.github/workflows/release.yml

```diff
@@ -32,20 +32,14 @@ jobs:
           force: ${{ github.event.inputs.force }}
           calver: true
       - name: Create new release in self-hosted Sentry
+        uses: getsentry/action-release@v1
         env:
-          SENTRY_DSN: 'https://19555c489ded4769978daae92f2346ca@self-hosted.getsentry.net/3'
           SENTRY_ORG: self-hosted
           SENTRY_PROJECT: installer
           SENTRY_URL: https://self-hosted.getsentry.net/
           SENTRY_AUTH_TOKEN: ${{ secrets.SELF_HOSTED_RELEASE_TOKEN }}
-          SENTRY_RELEASE: ${{ github.event.inputs.version }}
-        run: |
-          curl -sL https://sentry.io/get-cli/ | bash
-
-          # Create new Sentry release
-          sentry-cli --auth-token $SENTRY_AUTH_TOKEN releases new -p $SENTRY_PROJECT $SENTRY_RELEASE
-          sentry-cli --auth-token $SENTRY_AUTH_TOKEN releases set-commits --auto $SENTRY_RELEASE
-          sentry-cli --auth-token $SENTRY_AUTH_TOKEN releases finalize $SENTRY_RELEASE
-
-          # Create new deploy for this Sentry release
-          sentry-cli --auth-token $SENTRY_AUTH_TOKEN releases deploys $SENTRY_RELEASE new -e production
+        with:
+          environment: production
+          version: ${{ env.RELEASE_VERSION }}
```

Comment:
set in the prepare-release action, see [here](https://github.com/getsentry/self-hosted/actions/runs/3256633323/jobs/5347332615)

Reply:
Using the prepare-release action is definitely the right way to go. However, the `RELEASE_VERSION` is set in the environment but if we are going to rely on it I think it might be better to change the `prepare-release` action to set it as an output. You can see an example of what the changes to the action would be here: https://docs.github.com/en/actions/creating-actions/creating-a-composite-action#creating-an-action-metadata-file

Reply:
I'd rather not touch the `prepare-release` action since I'm not sure exactly where else it's used. Instead, I split up the  release job into two parts

1. Release a new version (`action-prepare-release`)
2. Create release on self-hosted dogfood instance (`action-release`)

I added an output for the 1st job that uses the release version from the `env.RELEASE_VERSION` that is set in `action-prepare-release`. The 2nd job uses the `needs` keyword to ensure the 1st job is run, and another `needs` keyword to ensure that the `release-version` output from the 1st job is available.

Reply:
let me know if that resolves your concerns!

Reply:
tested the run of the github action here:
https://github.com/getsentry/self-hosted/actions/runs/3292567306